app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+.body {
+  font-family: 'Open Sans';
+  margin: 0;
+}
+
+.main-section {
+  align-items: center;
+  border-top: 10px $orange solid;
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 50px 0;
+}
+
+.vertical-separator {
+  background: linear-gradient($light-grey, $orange, $light-grey );
+  content: '';
+  height: 400px;
+  margin: 0 40px 0 50px;
+  width: 3px;
+}
+
+.features {
+  align-items: center;
+  background-color: $light-grey;
+  display: flex;
+  justify-content: space-around;
+  margin: 0;
+  padding: 60px 0;
+  position: relative;
+
+  &:before {
+    background-image: image-url('triangle.png');
+    position: absolute;
+    top: -12px;
```

Comment:
you should set width 100% since this has no content, you should also add the `content: ''` for this to render.